lang/python/python-websockets/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2018-2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-websockets
+PKG_VERSION:=10.0
+PKG_RELEASE:=1
```

Comment:
any objections against $(AUTORELEASE) ?
also fine to leave this as-is

Reply:
No, I just forgot about it. Thank you for noticing this :+1: